homeassistant/components/reolink/button.py

```diff
@@ -95,6 +111,18 @@ class ReolinkButtonEntityDescription(
     ),
 )
 
+HOST_BUTTON_ENTITIES = (
+    ReolinkHostButtonEntityDescription(
+        key="reboot",
+        name="Reboot",
+        icon="mdi:restart",
```

Comment:
```suggestion
        device_class=BinarySensorDeviceClass.RESTART,
```

Reply:
I think you mean ButtonDeviceClass.RESTART, thanks!
was not aware of that one.